pkgs/development/node-packages/composition.nix

```diff
@@ -6,12 +6,12 @@
 
 let
   nodeEnv = import ./node-env.nix {
-    inherit (pkgs) stdenv python2 util-linux runCommand writeTextFile;
+    inherit (pkgs) stdenv python2 utillinux runCommand writeTextFile;
```

Comment:
Please revert all those changes. Also node2nix is currently broken and does not eval on master.

Reply:
If I understand the situation correctly it may make sense to hold off on merging anything touching nodePackages until the node2nix update is complete. I would then think I could re-run node2nix without producing these diffs.

Reply:
Yeah, kinda. It is not only this change but stdenv.lib to lib migration which also broke some things.